GunShipAlpha/src/Game.java

```diff
@@ -14,6 +14,7 @@ public class Game extends Canvas implements Mediator {
     private int waveNumber = 1;
     private int explosionCounter = 0;
     private ArrayList<ExplosionParticle> ongoingExplosions = new ArrayList<>();
```

Comment:
haha, for some reason i just love the name of this variable "ongoingExplosions".